src/styles/sign-up-page.scss

```diff
@@ -226,6 +226,16 @@
       font-weight: bold;
       letter-spacing: 3px;
     }
+
+    .closeIcon {
```

Comment:
```suggestion
    .close-icon {
```
Remember, all css class names should be written `in-this-format`

Reply:
It´s done!!